docs/jira.md

```diff
@@ -49,6 +49,10 @@ parent='KEY-XX'
 customfield_XXXXX='Custom field text'
 ```
 
+Type allows these values : 'Task', 'Sub-task'
```

Comment:
I think we should delete line 52, because there are a lot more values that Type could be: Doc, Improvement, Epic, Bug, etc.